Multiples focus

cesidio opened the pull request.

dan-palmer commented:
@chriswood123 LGTM ready to :ship:?

@gtvj if Chris is happy with this tomorrow then please merge.

chriswood123 commented:
Yep, ready to merge.

cesidio committed:
Introduce javascript to focus on multiples

cesidio committed:
Add jasmine gem with spec on anchor focus

This commit includes configuring jasmine to run on circle
& as part of plain 'rake'.

dan-palmer commented:
The jasmine specs should be running on circle too.
